File: src/main/java/com/lawencon/elearn/dto/examcollection/ExamCollectionInsertReqDto.java
package com.lawencon.elearn.dto.examcollection;

public class ExamCollectionInsertReqDto {
	private Long classDtlId;
	private Long examScheduleId;
	private String fileCode;
	private String fileExt;
	public Long getClassDtlId() {
		return classDtlId;
	}
	public void setClassDtlId(Long classDtlId) {
		this.classDtlId = classDtlId;
	}
	public Long getExamScheduleId() {
		return examScheduleId;
	}
	public void setExamScheduleId(Long examScheduleId) {
		this.examScheduleId = examScheduleId;
	}
	public String getFileCode() {
		return fileCode;
	}
	public void setFileCode(String fileCode) {
		this.fileCode = fileCode;
	}
	public String getFileExt() {
		return fileExt;
	}
	public void setFileExt(String fileExt) {
		this.fileExt = fileExt;
	}
}

File: frontend/src/app/dto/exam-collection/exam-collection.insert.req.dto.ts
export class ExamCollectionInsertReqDto {
    private classDtlId! : number
	private examScheduleId! : number
	private fileCode! : string
	private fileExt! : string

    public get _classDtlId(): number {
        return this.classDtlId;
    }

    public set _classDtlId(_classDtlId: number) {
        this.classDtlId = _classDtlId;
    }

    public get _examScheduleId(): number {
        return this.examScheduleId;
    }

    public set _examScheduleId(_examScheduleId: number) {
        this.examScheduleId = _examScheduleId;
    }

    public get _fileCode(): string {
        return this.fileCode;
    }

    public set _fileCode(_fileCode: string) {
        this.fileCode = _fileCode;
    }

    public get _fileExt(): string {
        return this.fileExt;
    }

    public set _fileExt(_fileExt: string) {
        this.fileExt = _fileExt;
    }

}